app/controllers/admins/users_controller.rb

```diff
@@ -17,22 +17,21 @@ def new
 
     def create
       @user = User.new(user_params)
-
+      @user.role_user! unless current_user.role_super_admin?
       if @user.save
-        redirect_to admins_user_url(@user), notice: ''
+        redirect_to admins_user_url(@user), notice: 'New user was successfully created.'
       else
         render :new, status: :unprocessable_entity
       end
     end
 
-    def edit
-    end
+    def edit; end
 
     def update
-      redirect_to admins_user_url(@user), notice: '' if last_super_admin_tries_to_update_its_role?
-
-      if @user.update(user_params)
-        redirect_to admins_user_url(@user), notice: '' 
+      if last_super_admin_tries_to_update_its_role?
```

Comment:
why did u removed guard clause? U can use multi line guard clause if there no enouth space for one-liner.

Reply:
Well i'm not sure that long string with flash message and condition in the end pass the rubocop.